src/components/tabBar/index.tsx

```diff
@@ -337,15 +309,11 @@ function createStyles({backgroundColor = DEFAULT_BACKGROUND_COLOR}) {
     },
     tabBar: {
       flex: 1,
-      height: DEFAULT_HEIGHT,
+      height,
       backgroundColor
     },
     shadowImage: {
       width: '100%'
     }
   });
 }
-
-TabBar.Item = TabBarItem;
-
-export default asBaseComponent<TabBarProps, {Item: typeof TabBarItem}>(TabBar);
```

Comment:
Same here, why would you want to stop using asBaseComponent?

Reply:
Done